src/configFirebase.js

```diff
@@ -0,0 +1,8 @@
+export const firebaseConfig = {
+  apiKey: 'AIzaSyCYHV8emihSZclfdX0XFdWRbhKwhSrnBP0',
+  authDomain: 'dims-17.firebaseapp.com',
+  projectId: 'dims-17',
+  storageBucket: 'dims-17.appspot.com',
+  messagingSenderId: '649779985987',
+  appId: '1:649779985987:web:03aaa1dab65e7a55c61ef6',
```

Comment:
You should not push to repository. All sensitive values must be kipped in `.env` file and ignored.

Reply:
Ok, thanks. Changed to `.env`